app/models/importers/content_items_by_organisation.rb

```diff
@@ -0,0 +1,20 @@
+module Importers
+  class ContentItemsByOrganisation
+    def run(slug)
+      organisation = Organisation.find_by(slug: slug)
```

Comment:
I would prefer a hard fail if the organisation is not found because:

1. It is a bit ackward to load the content items of an organisation that does not exist. 
2. The organisation has other attributes than the slug. It doesn't feel right to me to have organisations only with a slug
3. This would allow the creation of an organisation when the user has a typo.